test/suites/dev/moonbase/test-storage-growth/test-precompile-storage-growth.ts

```diff
@@ -74,6 +75,9 @@ describeSuite({
         const balBefore = await context.viem().getBalance({ address: FAITH_ADDRESS });
         const { abi: ierc20Abi } = fetchCompiledContract("IERC20");
 
+        // old value 91_088n
+        const expectedGas = 91_472n;
```

Comment:
Do you know why is this happening? From what I understand, both test scenarios should rely on the same effective gas (which in this case is the pov one). Not sure if this is a problem.

Reply:
It was because of this change: https://github.com/moonbeam-foundation/frontier/blob/18aa99b365d32a1524b4e7591f3797378c11fb0f/primitives/evm/src/lib.rs#L65